docs/maintainers/repository-structure.md

```diff
@@ -112,6 +112,7 @@ You can combine labels to find the issues you'd like, e.g. to [list all the issu
 [github-issues-status-help-wanted]: https://github.com/exercism/v3/issues?q=is%3Aissue+is%3Aopen+label%3Astatus%2Fhelp-wanted
 [github-issues-status-in-progress]: https://github.com/exercism/v3/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fin-progress
 [github-issues-status-draft]: https://github.com/exercism/v3/issues?q=is%3Aopen+is%3Aissue+label%3Astatus%2Fdraft
+[github-issues-status-wontfix]: https://github.com/exercism/v3/issues?q=is%3Aissue+label%3Astatus%2Fwontfix+is%3Aclosed
```

Comment:
Why do we need this (ie what's wrong with the one on the line below)

Reply:
If you view the file in the current state, the link already exists, it's just not linking to anywhere: 
![grafik](https://user-images.githubusercontent.com/20866761/97441339-e8336280-1928-11eb-8dd3-beb930b0ec99.png)

Reply:
Should the LOC below this be removed then?

Reply:
I completely missed that. oops.

Reply:
fixed

Reply:
Sweet! :)